app/models/analytics-event.ts

```diff
@@ -2,6 +2,6 @@ import Model from '@ember-data/model';
 import { attr } from '@ember-data/model';
 
 export default class AnalyticsEvent extends Model {
-  @attr('string') name;
-  @attr() properties;
+  @attr('string') declare name: string;
+  @attr() declare properties: { key: string; value: string };
```

Comment:
@libmartinito this type is incorrect, this would only match something like `{key: "a", value: "b"}`